<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpacjaTeleClientClass</class>
 <widget class="QMainWindow" name="SpacjaTeleClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SpacjaTeleClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ClientName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Åšrodek</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IpAddr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="MicLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mic is On</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoomLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Room #</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ChannelSpin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>42</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="IpLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Addr: </string>
    </property>
   </widget>
   <widget class="QLabel" name="LiveLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReadyLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MicSig">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>132</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="liveSig">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadySig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="spacjateleclient.qrc"/>
 </resources>
 <connections/>
</ui>
